doc/user/content/third-party/supported-tools.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Supported Tools Overview"
+description: "Get details about third-party tool support with Materialize"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+Materialize fully supports the following third-party tools:
+
+| Tool | Purpose |
+|------|---------|
+| [Docker](docker.md) | Easily deploy Materialize and other required infrastructure.
```

Comment:
I believe these links can just be to e.g. `[Docker](docker)`; seeing the `.md` extension isn't quite right, I don't think

Reply:
Removing the `.md` did not work for me locally, unfortunately! Not sure if there's something I'm setting up incorrectly here.

Reply:
The answer was to turn it into this: `[Docker](/third-party/docker)`!